caproto/threading/client.py

```diff
@@ -489,6 +493,7 @@ def search(self, results_queue, names, *, timeout=2):
 
         new_id = self.new_id
         unanswered_searches = self.unanswered_searches
+        pv_name_logger = logging.getLogger(f'caproto.bcast.search.{name}')
```

Comment:
I do not think `name` is defined here. This would have to be done inside a loop over `names`.